custom/yeksi_naa_reports/sqldata.py

```diff
@@ -190,49 +190,55 @@ class AvailabilityData(VisiteDeLOperateurDataSource):
 
     def calculate_total_row(self, rows):
         total_row = ['Availability (%)']
+        total_nominator = 0
```

Comment:
Can you change this to `total_numerator`

Reply:
Sure, good catch.